ego/templates/deployment.yaml

```diff
@@ -79,6 +79,21 @@ spec:
               value: {{ .Values.appConfig.swaggerHost }}
             - name: SWAGGER_BASEURL
               value: {{ .Values.appConfig.swaggerBaseUrl }}
+            - name: APITOKEN_DURATIONDAYS
+              value: {{ .Values.appConfig.apiTokenDurationDays }}
+            - name: JWT_SECRET
+              valueFrom:
+                secretKeyRef:
+                  name: {{ template "ego.fullname" . }}
+                  key: jwt-secret
+            - name: JWT_DURATIONMS
+              value: {{ .Values.appConfig.jwt.durationMs }}
+            - name: REFRESHTOKEN_DURATIONMS
+              value: {{ .Values.appConfig.refreshToken.durationMs }}
+            - name: REFRESHTOKEN_COOKIEISSECURE
+              value: {{ .Values.appConfig.refreshToken.cookieIsSecure }}
+            - name: REFRESHTOKEN_DOMAIN
+              value: {{ .Values.appConfig.host }}
 {{- if .Values.extraEnv }}
```

Comment:
this if block should enable passing env without updating charts under `extraEnv` actually, is this still necessary?

Reply:
although i dont like this `extraEnv`, it is neccessary especially we are in production and if we need to disable/enable something quickly without updating the chart. Its more of a last resort i guess